src/components/layout/footer/common/style.js

```diff
@@ -89,6 +90,11 @@ export const LinksCol = styled(Flex)`
     :last-child {
         margin-right: 0;
     }
+
+    @media (min-width: 993px) and (max-width: 1022px) {
```

Comment:
```suggestion
    @media (min-width: 993px) and (max-width: 1050px) {
```

Reply:
I think you can set it earlier at like 1050px as the columns already look imbalanced around that screen width.